src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1667,8 +1667,10 @@ public void load() {
                 File jobCredentials = new File(job.getRootDir(), "subversion.credentials");
```

Comment:
It is a bit strange situation to have `subversion.credentials` file but the Job is configuring using a Git repo :tired_face: